react-native-libraries.json

```
@@ -1,4 +1,20 @@
 [
+  {
+  "githubUrl": "https://github.com/getsettalk/rn-build-version",
+  "npmPkg": "rn-build-version",
+  "ios": false,
+  "android": true,
+  "web": false,
+  "windows": false,
+  "macos": false,
+  "tvos": false,
+  "visionos": false,
+  "expoGo": false,
+  "fireos": false,
+  "unmaintained": false,
+  "dev": true,
+  "newArchitecture": true
+},
```

Hey @getsettalk, thanks for adding a new package to the directory! 🎉 

Can you remove all `false` values, and move the entry at the end of file? We use the order from data JSON for "Recently Added" sort option on the website, see:
* https://github.com/react-native-community/directory?tab=readme-ov-file#how-do-i-add-a-library

```suggestion
  {
  "githubUrl": "https://github.com/getsettalk/rn-build-version",
  "android": true,
  "dev": true,
  "newArchitecture": true
},
```

---

yes sure